### PR TITLE
Put variables in constants.py in alphabetical order

### PR DESCRIPTION
**Pull request by bourque**
This PR re-orders `constants.py` to define variables in alphabetical order (except for those variables that depend on the definition of other variables).  I thought this might be a good idea now that we have a significant amount of variables defined in this module.

### CODE REVIEW

**Comment by bourque** [comment]
@hover2pi @bhilbert4 @Johannes-Sahlmann @laurenmarietta Could one of you review this?  Just a super simple PR that re-orders things in `constants.py`

File: jwql/utils/constants.py [hunk 1]
```diff
@@ -7,23 +7,40 @@
 
 Use
 ---
-    To print the JWST instrument names do:
+    This variables within this module are inteded to be directly
```
Comment (laurenmarietta) [inline jwql/utils/constants.py]: Typo here: "intended"

File: jwql/utils/constants.py [hunk 2]
```diff
@@ -20,45 +20,58 @@
     ``utils.py``
 """
 
+# Possible suffix types for AMI files
 AMI_SUFFIX_TYPES = ['amiavg', 'aminorm', 'ami']
```
Comment (Johannes-Sahlmann) [inline jwql/utils/constants.py]: e.g. here:
`NIRISS_AMI_SUFFIX_TYPES`.

File: jwql/utils/constants.py [hunk 3]
```diff
@@ -20,45 +20,58 @@
...
 
+# Defines the x and y coordinates of amplifier boundaries
 AMPLIFIER_BOUNDARIES = {'nircam': {'1': [(0, 0), (512, 2048)], '2': [(512, 0), (1024, 2048)],
                                    '3': [(1024, 0), (1536, 2048)], '4': [(1536, 0), (2048, 2048)]}
                         }
 
+# Possible suffix types for coronograph exposures
 CORONAGRAPHY_SUFFIX_TYPES = ['psfstack', 'psfalign', 'psfsub']
```
Comment (Johannes-Sahlmann) [inline jwql/utils/constants.py]: is this only for NIRCam?

File: jwql/utils/constants.py [hunk 4]
```diff
@@ -20,45 +20,58 @@
...
 
+# NIRCam subarrays that use four amps for readout
 FOUR_AMP_SUBARRAYS = ['WFSS128R', 'WFSS64R']
```
Comment (Johannes-Sahlmann) [inline jwql/utils/constants.py]: change name to `NIRCAM_AMP_SUBARRAYS`

File: jwql/utils/constants.py [hunk 5]
```diff
@@ -88,13 +101,17 @@
...
 
+# NIRCam long wavelength detector names
 NIRCAM_LONGWAVE_DETECTORS = ['NRCA5', 'NRCB5']
 
+# NIRCam short wavelength detector names
 NIRCAM_SHORTWAVE_DETECTORS = ['NRCA1', 'NRCA2', 'NRCA3', 'NRCA4',
                               'NRCB1', 'NRCB2', 'NRCB3', 'NRCB4']
 
+# NIRCam subarrays that use either one or four amps
 SUBARRAYS_ONE_OR_FOUR_AMPS = ['SUBGRISMSTRIPE64', 'SUBGRISMSTRIPE128', 'SUBGRISMSTRIPE256']
```
Comment (Johannes-Sahlmann) [inline jwql/utils/constants.py]: `NIRCAM_SUBARRAYS_ONE_OR_FOUR_AMPS`

**Comment by bourque** [comment]
Good suggestions @Johannes-Sahlmann, I'll make the appropriate changes.

**Comment by bourque** [comment]
@Johannes-Sahlmann I've adopted your suggestions, does this look good to you?

**Comment by bourque** [comment]
I have adopted the suggestions from @Johannes-Sahlmann, so I think this is good to go now; will merge pending successful Jenkins builds.

**Comment by bourque** [comment]
I'm not sure why `test_get_pipeline_steps` is failing: https://ssbjenkins.stsci.edu/blue/organizations/jenkins/STScI%2Fjwql/detail/PR-378/6/pipeline/22#step-54-log-34

This test passes for me locally just fine.

**Comment by bourque** [comment]
Ahh, I see now that there is already an issue for this (#398)

**Comment by laurenmarietta** [comment]
I'll observe that `test_get_pipeline_steps()` failing for Py 3.5 isn't actually the culprit for failing the Jenkins builds - it's `test_gaussian1d_fit` that's doing that here. 

I've seen this happen a few times now - I assume there's something going on with randomness so that sometimes the `np.isclose` assertion is True and other times False. I'll make a new issue for that.

**Pull request merged**